company_website/static/main_page/partials/_hamburger_menu.sass

```diff
@@ -0,0 +1,68 @@
+@import "constants"
+
+.hamburger
```

Comment:
Same here about class wrapper

Reply:
`.hamburger-active` is a class which I add after click on `.hamburger` class so they are on the same level.